### PR TITLE
adding table of run states

### PR DESCRIPTION
**Pull request by bcsherma**
Adding in a table of run states

Changing "activating" launch agent to "starting" launch agent

## Checklist
Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply. 

- [x] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [x] Build (`yarn build`) was run locally and successfully without errors or warnings.
- [x] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/launch/add-jobs-to-queue.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ import TabItem from '@theme/TabItem';
 
 # Add jobs to your queue
 
-Add W&B Jobs to your Launch Queue. Launch queues possess one or more W&B Jobs that are executed when you start a W&B Launch agent. A launch queue has a compute resource associated to that queue. Launch queues are entity-scoped. This means that anyone who has access to the entity can use the queue to launch jobs. 
+Launch queues possess one or more W&B Jobs that are executed when you start a W&B Launch agent. A launch queue has a compute resource associated to that queue. Launch queues are entity-scoped. This means that anyone who has access to the entity can use the queue to launch jobs.
```
Comment (KyleGoyette) [inline docs/guides/launch/add-jobs-to-queue.md]: They don't necessarily contain one or more jobs, that wording feels a little off for me.

I also don't love "entity-scoped" could we say "user or team" scoped?

File: docs/guides/launch/add-jobs-to-queue.md [hunk 2]
```diff
@@ -105,14 +101,26 @@ Each run listed corresponds to a job that was was added to that queue. For examp
...
-- **Creation date**: The time stamp when the queue was created.
-- **Start time**: The timestamp when the job started.
-- **Duration**: Time, in seconds, it took to complete the job’s run.
+   - **Run**: The name of the W&B Run assigned to that job.
+   - **Job ID**: The name of the job. See [Job naming conventions](create-job#job-naming-conventions) page for information on the default naming assigned to a job.
+   - **Project**: The name of the project the run belongs to.
+   - **Status**: The status of the queued run. 
+   - **Author**: The W&B entity that created the run.
+   - **Creation date**: The time stamp when the queue was created.
+   - **Start time**: The timestamp when the job started.
```
Comment (KyleGoyette) [inline docs/guides/launch/add-jobs-to-queue.md:108]: timestamp or time stamp, make the two lines the same

**Review by KyleGoyette — COMMENTED** [review]
A couple small changes

File: docs/guides/launch/add-jobs-to-queue.md (hunk 2 above)
Comment (bcsherma) [reply docs/guides/launch/add-jobs-to-queue.md:108]: good call

**Review by KyleGoyette — APPROVED** [review]
Should we remove the commented code? or leave it in?

**Comment by bcsherma** [comment]
> Should we remove the commented code? or leave it in?

@KyleGoyette not sure what you're referring to

**Review by KyleGoyette — APPROVED** [review]
They were not comments, the indents confused me.

**Comment by tristan-spaulding2** [comment]
What else is pending before we can merge this?

**Comment by bcsherma** [comment]
I can merge, looks like I need to resolve a few conflicts first

**Pull request merged**